iocore/net/quic/QUICPacketReceiveQueue.cc

```diff
@@ -63,7 +63,7 @@ QUICPacketReceiveQueue::enqueue(UDPPacket *packet)
 }
 
 QUICPacketUPtr
-QUICPacketReceiveQueue::dequeue(QUICPacketCreationResult &result)
+QUICPacketReceiveQueue::dequeue(QUICPacketCreationResult &result, bool hs_completed)
```

Comment:
Maybe I don't understand the issue and/or draft-13, but passing `hs_completed` doesn't seem a right direction. `_pn_protector` or  `_packet_factory ` should be able to check whether handshake is completed and they return some error if it's not completed (or they are not ready to decrypt the packet).